### PR TITLE
Add unreported exit code

### PR DESCRIPTION
**Pull request by keitharobertson**

### CODE REVIEW

File: python/sdk/prelude_sdk/models/codes.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ class ExitCode(Enum):
     MISSING = -1
     UNKNOWN_ERROR = 1
     MALFORMED_TEST = 2
+    UNREPORTED = 3
```
Comment (makk94) [inline python/sdk/prelude_sdk/models/codes.py:66]: and add to ERROR state down around line 130

**Review by makk94 — APPROVED** [review]

**Comment by makk94** [comment]
make sure FE is aware of this so they can update their sdk too

**Comment by makk94** [comment]
also update our docs https://docs.preludesecurity.com/docs/understanding-results

**Comment by stephanwampouille** [comment]
will this be going in before the threats deployment?

**Comment by keitharobertson** [comment]
> will this be going in before the threats deployment?

That's the plan unless that causes a problem for you guys

**Comment by stephanwampouille** [comment]
nope that's fine!

**Pull request merged**